source/security/private-endpoints.txt

```diff
@@ -252,3 +325,49 @@ You can toggle this setting in three ways:
 - In the Admin API, call the :admin-api-endpoint:`Toggle Non-VPC Requests
   <operation/allNonVpcClientRequests>` with a boolean value (``true`` to remove
   the default restriction, or ``false`` to reinstate it.)
+
+Definitions
+-----------
+
+There are three kinds of Private Endpoints:
+
+- Legacy: Endpoints that use ``realm.mogodb.com``
+- General: TODO: Add definition
+- Sync: TODO: Add definition
```

Comment:
```suggestion
- Legacy: Endpoints that use the legacy ``realm.mongodb.com`` domain. Apps with legacy endpoints must migrate their endpoints to use the new domain.
- General: Endpoints that use ``services.cloud.mongodb.com`` for Client HTTP requests.
- Sync: Endpoints that use ``services.cloud.mongodb.com`` for Sync WebSocket connections.
```

Reply:
Thank you for filling in the gaps here. 😄